### PR TITLE
MultiViewer.MultiViewerforF1 version 1.15.1

### PR DESCRIPTION
**Pull request by TheCodingBanana**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/103811)

### CODE REVIEW

File: manifests/m/MultiViewer/MultiViewerforF1/1.15.1/MultiViewer.MultiViewerforF1.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: MultiViewer.MultiViewerforF1
+PackageVersion: 1.15.1
+PackageLocale: en-US
+Publisher: MultiViewer
+PackageName: MultiViewer for F1
+License: MIT Licens
```
Comment (Trenly) [inline manifests/m/MultiViewer/MultiViewerforF1/1.15.1/MultiViewer.MultiViewerforF1.locale.en-US.yaml:9]:
```suggestion
License: MIT
```

**Review by Trenly — CHANGES_REQUESTED** [review]
IMG_9928

Silent switches may need to be specified

**Comment by stephengillie** [comment]
> IMG_9928
> 
> Silent switches may need to be specified

Could you post the setup log? I couldn't reproduce this in a Windows 11 VM.

File: manifests/m/MultiViewer/MultiViewerforF1/1.15.1/MultiViewer.MultiViewerforF1.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: MultiViewer.MultiViewerforF1
+PackageVersion: 1.15.1
+Installers:
+- Architecture: x86
```
Comment (stephengillie) [inline manifests/m/MultiViewer/MultiViewerforF1/1.15.1/MultiViewer.MultiViewerforF1.installer.yaml:7]:
```suggestion
- Architecture: x64
```

**Pull request closed without merging**